content/reference/endpoints/rental_cancelation_policy_items.md

```diff
@@ -12,9 +16,11 @@
   <div class="tab-pane active" id="public" markdown="1">
 Name               | Type    | Read/Write | Description
 -------------------|---------|------------|------------
-id                 | Integer | Read       | Rental cancelation policy item's id.
+account                   | Integer | Read       | Account's ID related to the Rental Cancelation Policy Item
+rental_cancelation_policy | Integer | Read       | Rental Cancelation Policy's ID related to the Rental Cancelation Policy Item
 -------------------|---------|------------|------------
-eligible_days      | Integer | Read       | Number of days before the check-in from which the penalty applies.
+id                 | Integer | Read       | Rental cancelation policy item's ID.
+eligible_days      | Integer | Read       | Number of days before the check-in from which the penalty applies (For example, if there is only a single Item defined for a Policy with eligible_days as 30, `penalty_percentage` doesn't apply before 30 days prior to check-in, only from 30 days and less). 
```

Comment:
We also have a uniqueness validation for eligible days, so users cannot create 2 policy items with the same day.
Might be good to mention, but not mandatory to me.

Reply:
@adamgrad But actually, it's not possible to create new items from API, so I would skip constraints.